src/util/strings.ts

```diff
@@ -0,0 +1,12 @@
+export const toCamelCaseStatus = (str = '') =>
```

Comment:
Didn't we have a helped already to do this similarly?

Reply:
@ovflowd 

I can't find any reference of the similar helper. Could you please share the location file ?